AppModelv2-WebApp-OpenIDConnect-DotNet/Controllers/ClaimsController.cs

```diff
@@ -15,18 +15,19 @@ public class ClaimsController : Controller
         /// <returns></returns>
         public ActionResult Index()
         {
-            var claimsPrincipalCurrent = System.Security.Claims.ClaimsPrincipal.Current;
+            var userClaims = User.Identity as System.Security.Claims.ClaimsIdentity;
+
             //You get the user’s first and last name below:
-            ViewBag.Name = claimsPrincipalCurrent.FindFirst("name").Value;
+            ViewBag.Name = userClaims?.FindFirst("name")?.Value;
```

Comment:
for the name of the claims, I was wondering if it would make sense to use the static members (strings) of
[System.Security.Claims.ClaimTypes](https://msdn.microsoft.com/en-us/library/system.security.claims.claimtypes%28v=vs.110%29.aspx)

@brentschmaltz : would you have a recommendation?

Reply:
@jmprieur - the claim's key name to obtain the user's first/last name is 'name', but there is no System.Security.Claims.ClaimTypes that points to this string. 

For example, System.Security.Claims.ClaimTypes.Name points to 'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name' instead

Reply:
Also, @andretms, as you are working on this sample, I've noticed that the README.md was not pointing to the right guided setup. It points to https://docs.microsoft.com/en-us/azure/active-directory/develop/guidedsetups/active-directory-windesktop

instead of https://docs.microsoft.com/en-us/azure/active-directory/develop/guidedsetups/active-directory-aspnetwebapp

Reply:
Thanks @jmprieur - good catch - I will update the readme as well and merge it